src/sonic-framework/tests/rebootbe_test.cpp

```diff
@@ -0,0 +1,420 @@
+#include "rebootbe.h"
+
+#include <gmock/gmock.h>
+#include <google/protobuf/util/json_util.h>
+#include <gtest/gtest.h>
+#include <unistd.h>
+
+#include <string>
+#include <thread>
+#include <vector>
+#include<iostream>
+
+#include "mock_reboot_interfaces.h"
+#include "reboot_common.h"
+#include "select.h"
+#include "status_code_util.h"
+#include "system/system.pb.h"
+#include "test_utils_common.h"
+#include "timestamp.h"
+
+namespace rebootbackend {
+
+#define ONE_SECOND (1)
+#define TWO_SECONDS (2)
+#define TENTH_SECOND_MS (100)
+#define HALF_SECOND_MS (500)
+#define ONE_SECOND_MS (1000)
+#define FIFTEEN_HUNDRED_MS (1500)
+#define TWO_SECONDS_MS (2000)
```

Comment:
This is a level of indirection that make it:
1. Hard to change going forward.
2. Does not enforce units (miliseconds, seconds), i.e. there is nothing that prevents you to write a `ONE_SECOND` in a parameter which expect miliseconds.

It is of high risk of introducing bugs for unsuspecting maintainers. Please remove it.

For Google code, consider using https://abseil.io/docs/cpp/guides/time

Reply:
There is a challenge in using absl. So, retaining this code